examples/scripts/logging-log4j-appender.cli

```diff
@@ -0,0 +1,4 @@
+/subsystem=logging/json-formatter=json:add(exception-output-type=formatted)
```

Comment:
I am wandering why this CLI script is needed? To have the subsystem in sync in case the server is reloaded? Otherwise  this bootable JAR JBoss logging should be fully configured after logging.properties file has been read? Question, after the logging.properties is rewritten by the subsystem, JBoss logging re-read it?

Reply:
What happens is the `logging.properties` is used at boot time only. The subsystem then creates the model based on the XML configuration and if changes are found in the current configuration then applies those changes. Once the subsystem re-configures the logging configuration the `logging.properties` is rewritten.

So this is required so the logging subsystem does not override the changes the `logging.properties` originally configures. Does that make sense?

Reply:
Ok, thank-you.